test/fixtures/suite/read_from_stdin/test.bats

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bats
+
+function cmd_using_stdin {
+    # Just reading from stdin
+    while read -r -t 1 foo; do
+        if [ "$foo" == "EXIT" ]; then
```

Comment:
This line is never reached. Maybe `test 3` could be something like
```bash
export -f cmd_using_stdin
run bash -c "echo 'EXIT' | cmd_using_stdin"
[ "$status" -eq 1 ]
```

Otherwise perhaps remove this block?

Reply:
Good point: A test where something is actually read from stdin was missing.

- Extracted command into own file to use with `bash -c`
- Change tests to use stdin and not use stdin
- Move from fixtures/suite to fixtures/bats where it is better suited
- Rebased on master